front-end/src/component/dashboard/dashboard.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import TodoForm from '../todo-form/todo-form';
+import * as todoActions from '../../actions/todo-actions.js';
+
+class Dashboard extends React.Component {
+
+  componentDidMount() {
+    this.props.todosFetch();
+  }
+
+  render() {
+    const { todos, todoCreate, todoUpdate, todoDelete } = this.props;
+    return (
+        <div className='dashboard'>
+          <h2> Todo: </h2>
+          <TodoForm
+          onComplete={todoCreate} {todoUpdate}
+          buttonText='Create Todo'
+        />
+        {
+          todos.map((todo) => {
+          return (
+              <div key={todo._id}>
+                <p>{todo.title}</p>
```

Comment:
render the TodoForm component in this map and pass in the `todoUpdate` function here as a property/attribute